README.md

```diff
@@ -97,6 +97,9 @@ Customizable variables can be created with the following properties:
 |step|If the value is not random, will be incremented each time by the value of step|
 |min|For random values defines it's minimum. Otherwise, will be the starting value|

```

Comment:
I wonder if we could make the min and max double values so that it works for ints and doubles?

Reply:
Would you add an overload for that that accepts ints?

Reply:
We would accept the min and max as double values.
We would have to convert to int for integer random variables.

Examples
a) `{"name": "Temp", "random": true, "max": 25, "min": 23}` --> min: 23, max 25
b) `{"name": "Temp", "random": true, "max": 25.9, "min": 23.5}` --> min: 23, max 25
c) `{"name": "Temp", "randomDouble": true, "max": 25.9, "min": 23.5}` --> min: 23.5, max 25.9